log/log_test.go

```diff
@@ -344,7 +345,15 @@ func TestStdoutFailure(t *testing.T) {
 
 func TestLogAtLevelEscapesNewlines(t *testing.T) {
 	var buf bytes.Buffer
-	w := &bothWriter{nil, 6, 0, clock.New(), &buf}
+	w := &bothWriter{sync.Mutex{},
+		nil,
+		&stdoutWriter{
+			stdout: &buf,
+			clk:    clock.NewFake(),
+			level:  6,
+		},
+		0,
```

Comment:
Do you intend this to be 0 (log to syslog at level 0) or -1 (don't log to syslog at all)? I think the latter makes more sense, given the nil syslog.Writer.